php/php.latte/test/unit/src/org/netbeans/modules/php/latte/parser/LatteParserErrorTest.java

```diff
@@ -43,18 +43,6 @@ public void testIssue245728_03() throws Exception {
         checkErrors("testfiles/parser/issue245728_03.latte");
     }
 
-    public void testIssue245728_04() throws Exception {
```

Comment:
I don't think that it's good to remove them completely. They should have been added for some reason. Someone might be able to fix them in the future (although I didn't check what they were tests for, I'm not sure whether we can fix). So, we should comment them out with the reason(e.g. test files were not donated) if you don't fix them.

Reply:
I returned methods, commented and added explanation as you suggested.